iscsi/targetd/ansible/initiator-playbook.yaml

```diff
@@ -5,19 +5,28 @@
     yum: 
       name: iscsi-initiator-utils
       state: latest
-  - name: Enable iscsid Service
-    service:
-      name: iscsid
-      enabled: yes
-      state: started
+  - name: Install device-mapper-multipath
+    yum:
+      name: device-mapper-multipath
+      state: latest
   - name: Write iscsi initiator name
     lineinfile:
       regexp: '^(InitiatorName=).*'
       dest: /etc/iscsi/initiatorname.iscsi
       line: '\1{{ iscsi_initiator_name }}'
       state: present
       backrefs: yes
-  - name: restart iscsid
+  - name: Write multipath conf file
+    copy:
+      src: multipath.conf
+      dest: /etc/multipath.conf
+  - name: Enable iscsid Service
     service:
       name: iscsid
-      state: restarted
+      enabled: yes
+      state: started
+  - name: enable multipathd service
+    service:
+      state: started
+      enabled: yes
+      name: multipathd
```

Comment:
@raffaelespazzoli there is a command called `mpathconf --enable` to enable and meet the requirements of multipath configuration. Have a check on it and adjust above accordingly if needed

Reply:
I don't understand the comment. the task "Write multipath conf file" takes care of the configuration.
which requirements do you think this approach is not meeting?

Reply:
@raffaelespazzoli I am not mandating it, I mention it if in case it helps to reduce any overhead on above process to meet the multipath requirement. It was a suggestion

Reply:
@humblec ok, then I think this piece of the code is good as it is now.